src/components/ButtonPanel.js

```diff
@@ -1,38 +1,40 @@
 import React from 'react'
 
+import '../styles/ButtonPanel.css'
+
 import Button from './Button'
 
 const ButtonPanel = () => {
   return (
-    <div id='button-panel'>
-      <div id='group-one' className='row'>
-        <Button name='AC' classNaming='generalBtn' />
-        <Button name='+/-' classNaming='generalBtn' />
-        <Button name=' %' classNaming='generalBtn' />
-        <Button name='÷' classNaming='generalBtn' />
+    <div id='button-panel' className='container'>
+      <div id='group-one' className='row bg-primary'>
+        <Button name='AC' color='##ffffff' wide={false} />
+        <Button name='+/-' color='##ffffff' wide={false} />
+        <Button name=' %' color='##ffffff' wide={false} />
+        <Button name='÷' wide={false} />
       </div>
       <div id='group-two' className='row'>
-        <Button name='7' classNaming='generalBtn' />
-        <Button name='8' classNaming='generalBtn' />
-        <Button name='9' classNaming='generalBtn' />
-        <Button name='X' classNaming='generalBtn' />
+        <Button name='7' color='##ffffff' wide={false} />
+        <Button name='8' color='##ffffff' wide={false} />
+        <Button name='9' color='##ffffff' wide={false} />
+        <Button name='X' wide={false} />
       </div>
       <div id='group-three' className='row'>
-        <Button name='4' classNaming='generalBtn' />
-        <Button name='5' classNaming='generalBtn' />
-        <Button name='6' classNaming='generalBtn' />
-        <Button name='-' classNaming='generalBtn' />
+        <Button name='4' color='##ffffff' wide={false} />
+        <Button name='5' color='##ffffff' wide={false} />
+        <Button name='6' color='##ffffff' wide={false} />
```

Comment:
When a property is boolean, you do not need to explicitly pass it false, not passing it at all would do the trick, so here the wide prop can be removed and for all other occurrences were it is set to false